README.md

```diff
@@ -2,37 +2,48 @@
 
 # cadvisor-integration
 ## Overview
-This tool designed to run inside of Kubernetes cluster. After installation it will auto discover cluster nodes and will query stats from them on time basis.
+This tool will auto discover kubernetes cluster nodes and send container and machine metrics to SignalFx. 
+You can deploy it to run in a single pod within a Kubernetes cluster where it will collect data from the cadvisor container resource usage and performance analysis agent (that is integrated into the kubelet binary) using cluster member permissions only. 
 ## Installation
-You may clone and build tool by yourself but the easiest way is to use prebuild docker image. Here is an example yaml file:
+**Step 1** - Create a deployment configuration based on the following example _cadvisor-signalfx.yaml_ file. **Note:** Change the &lt;API_TOKEN&gt; based on your SignalFx account and the &lt;CLUSTER_NAME&gt; based on what you want to call this kubernetes cluster.
 
 	apiVersion: v1
-	kind: Pod
+	kind: ReplicationController
 	metadata:
-	  name: "test-scrapper"
-	  labels:
-	    app: "test-scrapper"
+	  name: "cadvisor-signalfx"
 	spec:
-	  containers:
-	  - name: scrapper
-	    image: "quay.io/signalfx/prometheustosfx:latest"
-	    env:
-	    - name: SFX_SCRAPPER_API_TOKEN
-	      value: <API TOKEN>
-	    - name: SFX_SCRAPPER_CLUSTER_NAME
-	      value: <CLUSTER NAME>
-	    - name: SFX_SCRAPPER_SEND_RATE
-	      value: 5s
+	  replicas: 1
+	  selector:
+	    app: "cadvisor-signalfx"
+	  template:
+	    metadata:
+	      name: "cadvisor-signalfx"
+	      labels:
+	        app: "cadvisor-signalfx"
+	    spec:
+	      containers:
+	      - name: "cadvisor-signalfx"
+	        image: "quay.io/signalfx/prometheustosfx:latest"
```

Comment:
yea .. I need to change the use of that name "prometheustosfx" also as the appname in the code.  how about if i treat that change separate than this one (since it involves changes to other files) and I'll make another PR later today?